eslint-plugin/vitest.config.ts

```diff
@@ -6,5 +6,6 @@ export default defineConfig({
 		coverage: {
 			reporter: ['text', 'json', 'html'],
 		},
+		globals: true,
```

Comment:
I think it is better not to expose globals, and instead to explicitly link the functions needed by the rule tester in `svelte-check-props.spec.ts`:
```ts
import {afterAll, describe, test} from 'vitest';
// ...
import {RuleTester} from '@typescript-eslint/rule-tester';

RuleTester.describe = describe;
RuleTester.it = test;
RuleTester.afterAll = afterAll;

describe('svelte-check-props', () => {
// ...
});
```
(and the same in `angular-check-props.spec.ts`)

Reply:
Great proposal, it has been added thanks to your proposed changes commit.